src/labels/CustomLabels.labels

```diff
@@ -4214,6 +4270,548 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+        <protected>true</protected>
+        <shortDescription>Page-level error message header when Advanced Mapping off</shortDescription>
+        <value>Enable Advanced Mapping</value>
+    </labels>
+    <labels>
+        <fullName>geErrorPageLevelFieldPermission1</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Page-level error for insufficient field access with fields listed</shortDescription>
+        <!-- {0} is a list. Name of fields e.g. npsp__Amount__c, npsp__Description__c -->
+        <value>To save changes to this template you must have permission to edit all of the fields in the form. Please remove the following fields: {0}</value>
+    </labels>
+    <labels>
+        <fullName>geErrorPageLevelFieldPermission2</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Page-level error for insufficient field access about exiting without saving</shortDescription>
+        <value>Alternatively, exit this form without saving and check your permissions on both the target object and on the NPSP Data Import object.</value>
+    </labels>
+    <labels>
+        <fullName>geErrorPageLevelMissingRequiredFields</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Page-level error for insufficient field access about exiting without saving</shortDescription>
+        <!-- Label should be followed by a list of fields e.g. npsp__Amount__c, npsp__Description__c -->
+        <value>You must include all required fields in your template. Review the source fields (NPSP Data Import object) for the following fields:</value>
+    </labels>
+    <labels>
+        <fullName>geErrorRequiredField</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Field-level error message for required field missing on NPSP Data Import object</shortDescription>
+        <value>Required field</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderBatchHeaderLeftCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header to select batch header fields</shortDescription>
+        <value>Select Batch Header Fields</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderBatchHeaderRightCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header to configure batch header fields</shortDescription>
+        <value>Configure Batch Header Fields</value>
+    </labels>
+    <labels>
+        <fullName>geHeaderError</fullName>
```

Comment:
@psadie-sfdo should this be a more generalized dev name?

Reply:
@melissabarber - Looks like I did have the 'common' version of this in there and just didn't clean this out. Updating!